bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/MainConfig.java

```diff
@@ -51,6 +51,7 @@ public MainConfig(Plugin plugin, String name) {
         yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_DISABLED_LANGUAGES, Collections.singletonList("your language iso here"));
         yml.addDefault("storeLink", "https://www.spigotmc.org/resources/authors/39904/");
         yml.addDefault("lobbyServer", "hub");
+        yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_ENABLE_HALLOWEEN, true);
```

Comment:
Default halloween on?

Reply:
I have defaulted the function to true but in the config anyone who wants can disable it.
If yuou want i can set that on false as default.